commands/activations.go

```diff
@@ -111,10 +112,12 @@ func RunActivationsLogs(c *CmdConfig) error {
 	if argCount > 1 {
 		return doctl.NewTooManyArgsErr(c.NS)
 	}
+	replaceFunctionWithAction(c)
+	augmentPackageWithDeployed(c)
 	if isWatching(c) {
-		return RunSandboxExecStreaming("activation/logs", c, []string{"last", "strip", "follow"}, []string{"function", "package", "limit"})
+		return RunSandboxExecStreaming("activation/logs", c, []string{"last", "strip", "follow", "deployed"}, []string{"action", "package", "limit"})
```

Comment:
The values passed to this function can be top file `consts` for better maintainability

Reply:
We can focus on the files in this PR alone. `functions.go` and `sandbox-extra.go` can also be updated to use consts. 

The string values don't look like they repeat. For now, we can continue to put the const values at the beginning of each file.

The other files I can look at and update accordingly .

Reply:
If the string values repeat, then a `commands/const.go` file can be filled with all the consts so that the command files do not have to borrow values from each other. I would suggest moving all the const definitions there.

Reply:
I pushed a change to implement the suggestion.   I notice, though, that the auto-checks appear stuck.
Hope we can converge soon.